src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
+	public static int blockNumber = 123;
+	public Block(int blockNumber){
+		Block.blockNumber = 123;
+	}
+	public  int getNumber(){
+		return blockNumber;
+	}
+}
+class Street{
+	public static String streetName = "abc";
+	public Street(String streetName){
+		Street.streetName = "abc";
```

Comment:
When you want to refer a variable in the class use the keyword this. i.e. this.streetName= streetName for line 72